firefox_ui_tests/remote/security/test_safe_browsing_notification.py

```diff
@@ -0,0 +1,120 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import time
+
+from marionette_driver import (
+    By,
+    expected
+)
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestSafeBrowsingNotificationBar(FirefoxTestCase):
+
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.test_data = [
+            # Phishing URL info
+            {
+                'button_label': 'safebrowsing.notAForgeryButton.label',
+                'report_page': 'www.google.com/safebrowsing/report_error',
+                'unsafe_page': 'https://www.itisatrap.org/firefox/its-a-trap.html'
+            },
+            # Malware URL object
+            {
+                'button_label': 'safebrowsing.notAnAttackButton.label',
+                'report_page': 'www.stopbadware.org',
+                'unsafe_page': 'https://www.itisatrap.org/firefox/its-an-attack.html'
+            }
+        ]
+
+        self.prefs.set_pref('browser.safebrowsing.enabled', True)
+        self.prefs.set_pref('browser.safebrowsing.malware.enabled', True)
+
+        # workaround for bug 1084289; in smaller windows, clipping causes X button to disapper
+        self.marionette.maximize_window()
```

Comment:
This is not doing what you expect. I would suggest we kill that line and disable the button test. See [bug 1139369](https://bugzilla.mozilla.org/show_bug.cgi?id=1139369)